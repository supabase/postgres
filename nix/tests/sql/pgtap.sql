begin;

select plan(1);

-- Run the tests.
select pass( 'My test passed, w00t!' );

-- Finish the tests and clean up.
select * from finish();

rollback;
