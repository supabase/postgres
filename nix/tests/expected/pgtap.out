begin;
select plan(1);
 plan 
------
 1..1
(1 row)

-- Run the tests.
select pass( 'My test passed, w00t!' );
             pass             
------------------------------
 ok 1 - My test passed, w00t!
(1 row)

-- Finish the tests and clean up.
select * from finish();
 finish 
--------
(0 rows)

rollback;
